### DISCUSSION TITLE
Vertical Scroll for collection types what for?

### DISCUSSION THREAD

**Discussion by CtFelix** [strapi/strapi#7207] [Legacy Topics]
Hi everyone
If i have enough of space for show all of collection types, why do no do it?
[Image](https://yadi.sk/i/oYe3QJVUiGzJpg)

**Comment by lauriejim** [comment]
@HichamELBSI @maevalienard any insight about that?

**Reply by soupette** [reply]
Well it's only for UI reasons and the same rule applies for the main left menu. When we will introduce the design system it will be easier for users to change these kind of settings.

**Reply by CtFelix** [reply]
> When we will introduce the design system it will be easier for users to change these kind of settings.
When it will be?

**Reply by soupette** [reply]
let's ask @Aurelsicoko

**Reply by maevalienard** [reply]
Not yet, that will probably be in a couple of months to get the time to set up everything properly :)

**Reply by CtFelix** [reply]
Thank you
